lib/inets/src/inets_app/inets.erl

```diff
@@ -110,6 +116,14 @@ start(Type) ->
 %% top supervisor.
 %% --------------------------------------------------------------------
 -doc(#{equiv => start/3}).
+-spec start(Service, ServiceConfig) -> Result when
```

Comment:
I think we should add some text about that we consider the stand-alone mode deprecated. It is bad!

Reply:
Like doc text or deprecation warning; if it's not already in place? What do you suggest?